src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-tabs/smb-cluster-tabs.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SmbClusterTabsComponent } from './smb-cluster-tabs.component';
+
+describe('SmbClusterTabsComponent', () => {
+  let component: SmbClusterTabsComponent;
+  let fixture: ComponentFixture<SmbClusterTabsComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [SmbClusterTabsComponent]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(SmbClusterTabsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
we can add a test for tab heading

Reply:
I have added a couple tests verifying the selected item and the rendering of the tabs.
I am not sure if any other tests are expected, let me know if you find anything else worth to add as I wasn't really sure what to unit test from here.